packages/elements-next/src/index.mdx

```diff
@@ -7,10 +7,105 @@ route: /
+- The docs should be styled the same as Elements - Elements docs should look like Elements!
+- There should be far fewer components. We have several different ways of doing notifications - there should be a single approach. See below for list of components to migrate.
+- HTML output should be next to the mounted React Component and the Rendered Component in the docs in all cases.
+- All variants of components should be visible at the top of the page in the generated props section. They should all be editable at the bottom, although only the basic component without variants should be rendered by default.
+- Where we export multiple components from the same file, we should export them as modules eg `<Headings.Subheading>`
+- Should work in Edge (latest version before Chromium), Edge Chromium, Firefox, Chrome, Safari
+- Should use Linaria `cx` for composing classes.
+
+### Components to migrate
+
+The following components to be built in v2. Will add tickets to each one as they are written.
+
+- Alert, Info and Helper to be combined into a single module with variant components
+- Button as a module with all variants including button group
+- Input as a module to include; Text, Number, Password, Number, Select, Image Upload (there are two of these, should be combined), Checkbox, Radio, DatePicker, MultiSelect (vanilla refactor), HTML Editor, File Input, Text Area
+- Dynamic links, tidied and refactored if required.
+- HTML Renderer
+- Helper guide - can be massively tidied and refactored
+- Headings - already have a start on this, to tidy up
+- Lists - to include UL, OL and Icon Lists
+- Section with super flexible variants to act as containers for any element
+- Container, for pages and top level elements - as per Section, super flexible with modifiers
+- Grid, Grid Items - should be based on native CSS Grid, basis 12 cols
+- Loader - inline and page centered variants
+- Maps - consider ditching React Wrappers for Google maps - it's trivial to load a map in a UseEffect hook and saves tonnes of JS
+- Menu - shoud be refactored to better support actions, icons and sub menus as a dropdown
+- Notification, Toast, ToastMessage, all combined - single module to notify users, simple implementation with variants. Current notification is the most flexible.
+- Modal, single, simple implementation
+- Pagination
+- Spreadsheet - tidy and refactor as required
+- Table - just a simple styled HTML table, we don't use any advanced functionality and if we need it, just use Spreadsheet
+- Form - to export Formik, to include form Headings and Helpers
+- Tabs
+- Progress bar
+- Wizard - should be refactored to be more flexible, we will be using this a lot.
+
+### Libraries to include
+
+#### Definitely
+
+- DayJS
+- Formik
+- Linaria
+- React Icons
+- React Datasheet
+- React Image Crop
+- React Transition Group
+- A much better looking and mobile friendly date picker
```

Comment:
The date picker we're using (react-datepicker) is fine to me, we can customize the style for better looking instead of using its default styles.
It also comes with a modal version that we can show to mobile users
![image](https://user-images.githubusercontent.com/22409039/92347412-3894fd80-f0fa-11ea-8a6c-350f4942ad38.png)

Reply:
@nphivu414  I do think it's really ugly so we need to spend some time styling it to be more "elements" if we keep it. I was not aware of the modal view, it is better definitely. Happy to keep the lib if we can improve the UX maybe worth looking at options though